diff_cover/__init__.py

```diff
@@ -1,5 +1,5 @@
 # This file is imported in setup.py and cannot use any external dependencies
 
-VERSION = "4.0.1"
+VERSION = "4.0.2"
```

Comment:
id prefer if you allowed me to handle this after merging. I like to handle all the packaging stuff together in a commit when im ready to release

Reply:
sure right I will revert this change - I used it just to make 100% I was using my local version of the upstream :)